Lib/ipaddress.py

```diff
@@ -1146,11 +1141,11 @@ class _BaseV4:
     """
 
     __slots__ = ()
-    _version = 4
+    version = _version = 4
```

Comment:
Is there a need for `_version` and `_max_prefixlen` after this change? Why not remove them entirely and use `version` and `max_prefixlen` instead?

Reply:
There should not be a need, they are _private.  If anything turns up during the 3.14 beta/rc period that wants them we can re-add them as properties.

Reply:
I've switched to the public version (pun not intended). I hope this doesn't make this harder to review.